### DISCUSSION TITLE
Whitelist support?

### DISCUSSION THREAD

**Discussion by sierraworks** [theengs/gateway#180] [Q&A]
I notice that OMG has the ability to supply a whitelist list on startup.  Before I add it to the gateway is it already there in some undocumented location?   Great app and runs great on a Rasp Pi W1 (so far).

**Comment by koenvervloesem** [comment]
There's currently no whitelist implemented in Theengs Gateway. So go ahead with your implementation and feel free to ask for assistance in this discussion thread.
